proto/cacheclient.proto

```diff
@@ -136,8 +131,7 @@ message _DictionaryIncrementRequest {
 }
 
 message _DictionaryIncrementResponse {
-  ECacheIncrementResult status = 1;
-  int64 value = 2;
+  int64 value = 1;
```

Comment:
For `DictionaryIncrement`, the behavior would be the same as `DictionarySet` where we will create the dictionary if it does not exist and add the field to it and initialize the field to 0. Correct? 

Also should the `amount` provided in the request and the `value` returned in the response have type `uint` instead of `int`?

Reply:
@cprice404 https://github.com/momentohq/client_protos/pull/88#pullrequestreview-1102826604

Reply:
re: uint - Michael's docs state that both can be negative, which seems reasonable to me.

Reply:
@anp13 Yes same behavior for `DictionarySet`.

`uint` vs `int64`: while the name of the command is `increment`, the semantics are `+=`. Thus positive, negative, and zero amounts are all valid.

For more details see the spec on notion and [C# documentation](https://github.com/momentohq/client-sdk-csharp/blob/main/src/Momento.Sdk/Incubating/SimpleCacheClient.cs#L240-L277)

Reply:
Ohh. So user can use this api to decrement the value as well? The name says "increment" but I guess technically you can use a negative integer to decrement the value as well. Got it.